pkg/proxy/metaproxier/meta_proxier.go

```diff
@@ -62,33 +64,47 @@ func (proxier *metaProxier) SyncLoop() {
 
 // OnServiceAdd is called whenever creation of new service object is observed.
 func (proxier *metaProxier) OnServiceAdd(service *v1.Service) {
-	if *(service.Spec.IPFamily) == v1.IPv4Protocol {
-		proxier.ipv4Proxier.OnServiceAdd(service)
+	// if ClusterIP is "None" or empty, or ExternalName service skip proxying
+	svcName := types.NamespacedName{Namespace: service.Namespace, Name: service.Name}
+	if utilproxy.ShouldSkipService(svcName, service) {
```

Comment:
Not sure the comment is needed... `ShouldSkipService` makes it pretty clear what's going on here

Reply:
removed